scripts/exgfs_nceppost_cpl.sh

```diff
@@ -436,65 +499,56 @@ do
        mv goesifile $COMOUT/${SPECIALFLIDX}f$fhr
 
     fi
-   # rm flxfile flxifile goesfile goesifile    
-    rm flxifile goesfile goesifile    
     fi
 # end of satellite processing
 
-##########################  GTG  start ##########################
-# process  gtg turbulence separately so that master pgb and
-# satellite look alike gets out in time.
-
-# process gtg turbulence only when fhr is every 3 hour between [3, 36]
-    if [[ $fhr -ge 3  &&  $fhr -le 36  && $((fhr%3)) -eq 0 && $GTGF = "YES" ]] ; then
-      export OUTTYP=${OUTTYP:-4}
-
-# specify output file name from chgres which is input file name to nceppost
-# if model already runs gfs io, make sure GFSOUT is linked to the gfsio file
-# new imported variable for global_nceppost.sh
-      export GFSOUT=${PREFIX}gfsio${fhr}
-
-      # gtg has its own configurations
-      cp $PARMpost/gtg.config.gfs gtg.config
-
-      if [[ $RUN = gfs && $GRIBVERSION = 'grib2' ]] ; then
-          export PostFlatFile=$PARMpost/postxconfig-NT-GFS-GTG.txt
-          export CTLFILE=${CTLFILEGTG:-$PARMpost/postcntrl_gfs_gtg.xml}
-
-#          export SIGINP=sigfile
-#          export NEMSINP=${NEMSINP:-nemsfile}
-#          export SFCINPUT=sfcfile
-#          export FLXINP=${FLXINP:-flxfile}
-          export FLXIOUT=flxifile
-          export PGBOUT=gtgfile
-          export PGIOUT=gtgifile
-          export FILTER=0
-          export IGEN=$IGEN_GFS
-
-          $POSTGPSH
-          export err=$?
-          err_chk
+##########################  WAFS start ##########################
+# Generate WAFS products on ICAO standard level.
+# Do not need to be sent out to public, WAFS package will process the data.
+    if [[ $WAFSF = "YES"  && $fhr -le 120 ]] ; then
+       if [[ $RUN = gfs && $GRIBVERSION = 'grib2' ]] ; then
+          export OUTTYP=${OUTTYP:-4}
+
+	  # Extend WAFS icing and gtg up to 120 hours
+          export PostFlatFile=$PARMpost/postxconfig-NT-GFS-WAFS.txt
+          export CTLFILE=$PARMpost/postcntrl_gfs_wafs.xml
+
+          # gtg has its own configurations
+          cp $PARMpost/gtg.config.gfs gtg.config
+          cp $PARMpost/gtg_imprintings.txt gtg_imprintings.txt
+
+          export PGBOUT=wafsfile
+          export PGIOUT=wafsifile
+
+          # WAFS data is processed:
+          #   hourly if fhr<=24
+          #   every 3 forecast hour if 24<fhr<=48
+          #   every 6 forecast hour if 48<fhr<=120
+	  if [  $fhr -le 24  ] ; then
```

Comment:
Indentation not consistent with previous

Reply:
This is copied from the NCEP post library -- it should just be linked in but we have to add one extra loop for the flux.1p00, therefore if the indentation should be fixed, it should be fixed in the NCEP Post library and copied in here.  This file shouldn't even exist in this repo, but it's the only work around we have for now.